web/themes/custom/pineapple/components/_hero-image.twig

```diff
@@ -24,8 +24,11 @@
       {% endif %}
       {% if labels %}
         <div class="hero-labels">
-          {% for item in labels %}
-            {{ item.value }}
+          {% for label in labels %}
```

Comment:
Consider using the word "item" instead of "label", looks odd doing `label.label`

Reply:
done